spec/board_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../lib/board'
+require_relative 'custom_matcher'
+
+RSpec.describe Board do
+    describe "#total_pieces" do
+    	it 'has a total of 32 pieces' do
+    		b = Board.new
+    		expect(b.total_pieces).to CustomMatcher.custom_matcher
```

Comment:
Your spec has the text `has a total of 32 pieces` - but this does not reflect in the actual test. Additionally the custom_matcher - does it result in 32 as required?